src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,19 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
 
+        Machine[] machines = new Machine[] {truck, bulldozer, excavator};
+
+        for (Machine machine : machines) {
+            machine.doWork();
+        }
+
+        for (Machine machine : machines) {
+            machine.stopWork();
+        }
+    }
```

Comment:
```suggestion
        Machine bulldozer = new Bulldozer();
        Machine[] machines = new Machine[] {truck, bulldozer, excavator};
        for (Machine machine : machines) {
            machine.doWork();
        }
        for (Machine machine : machines) {
            machine.stopWork();
        }
    }
```

I dont think we need this redundant empty lines